main.go

```diff
@@ -123,7 +127,15 @@ func main() {
 		apiroots[i] = path.Join(root, appname, apiversion)
 	}
 
-	srv, err := NewServer(addr, apiroots, db)
+	msgChannel := make(chan []byte, 100)
```

Comment:
Is there a reason you hard-code the channel buffer to 100? That seems like it should either be unbuffered, or set via a flag value.

Reply:
We did it in ingress, but I'm not sure of the reasoning there either. Perhaps to just limit the memory usage of the pod if something got hung up?